rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/common/CommonRest.java

```diff
@@ -150,6 +146,28 @@ public UserData currentUserData(String sessionId) {
         return answer;
     }
 
+    @Override
+    public Map<String, Boolean> checkPermissions(String sessionId, List<String> methods) throws RestException {
+        Scheduler scheduler = checkAccess(sessionId);
+        if (methods == null || methods.isEmpty()) {
+            return new HashMap<>();
```

Comment:
How about returning `Collections.emptyMap()` to save a little memory and returning an immutable object